src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,7 +11,7 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
```

Comment:
Missing accessor method for feedbackToUser, and missing modification in TextUI.java to call the accessor method.